### PR TITLE
Update NavBarContainer.js

### PR DESCRIPTION
**Pull request by davidLeonardi**
Added support for hiding navbar per route
Style improvements

### CODE REVIEW

**Comment by MikaelCarpenter** [comment]
is the extra indentation on the if statements on purpose?

**Comment by davidLeonardi** [comment]
Nope. let me fix that.

**Comment by davidLeonardi** [comment]
I wish atom had a better way of switching indentation methods per file. Currently it's global, and i use tabs. I try to remedy all these misindentations manually, but this slipped through. sry :(

File: components/NavBarContainer.js [hunk 1]
```diff
@@ -41,22 +40,51 @@ var NavBarContainer = React.createClass({
...
-            route={this.state.previousRoute}
-            backButtonComponent={this.props.backButtonComponent}
-            rightCorner={this.props.rightCorner}
-            titleStyle={this.props.titleStyle}
-            willDisappear="true" /> )
-        }
+    if (this.props.currentRoute.trans) {
+      trans = {backgroundColor: 'transparent'};
+    }
+    else {
```
Comment (mikefowler) [inline components/NavBarContainer.js]: This should probably be consistent with the One True Brace in the rest of the code.

Comment (davidLeonardi) [inline components/NavBarContainer.js]: indeed.

**Comment by davidLeonardi** [comment]
Mikael, this has been fixed. Let me know if something else pops up.

**Pull request merged**